src/ralph/operations/api.py

```diff
@@ -13,9 +13,17 @@ class Meta:
         depth = 1
 
 
+class OperationStatusSerializer(RalphAPISerializer):
+
+    class Meta:
+        model = OperationStatus
+        depth = 1
+
+
 class OperationSerializer(RalphAPISerializer):
     type = OperationTypeSerializer()
     assignee = SimpleRalphUserSerializer()
+    status = OperationStatusSerializer()
```

Comment:
That's an interesting option actually. However, I would like to discuss this topic a little in order to find the best solution. My concern is caused by the fact that now statuses are not hardcoded but stored in DB and allow dynamic management. Perhaps slugifying will bring more issues to API consumers than comfort. Say, someone wants to store an operation in an external data store and then use its status in order to find other operations. If the status is slugified, it won't be possible, if the name of the status was changed. So, I would vote for exposing statuses as structures. @mkurek, @vi4m, your opinions?